examples/demo.js

```diff
@@ -1,39 +1,48 @@
 import React, { Component } from 'react'
 import clsx from 'clsx'
 import Villain from '@/index.js'
-import {version} from '../package.json';
+import { version } from '../package.json'
 import './demo.css'
+import ToggleThemeButton from './ToggleThemeButton'
 
 class Demo extends Component {
-
   state = {
     file: '/build/testFile/Example-archive.zip',
     options: {
       preview: 0,
-      theme: {type: 'select', value: 'Light', options: ['Dark', 'Light']},
+      theme: { type: 'select', value: 'Light', options: ['Dark', 'Light'] },
       mangaMode: false,
       allowFullScreen: true,
       autoHideControls: false,
       allowGlobalShortcuts: false,
-    }
+    },
+    theme: 'light',
   }
 
   constructor(props) {
     super(props)
   }
 
+  toggleTheme = () => {
+    const theme = this.state.theme === 'light' ? 'dark' : 'light'
+    document.documentElement.classList.add('color-theme-in-transition')
```

Comment:
why do we need this ?

Reply:
Actually I used a different approach on adding a theme . Instead of using the old React context and all, I tried using css variables. Please do let me know if this is to be made using context only

Reply:
I don't know what you mean by the `old react context`. We are not using the context api on the example ( demo component ) :confused: 

Anyway please don't modify the DOM like that, just use react.

Reply:
See: https://github.com/btzr-io/Villain/pull/168#discussion_r339259558